specification/quantum/Microsoft.Quantum.Management/models.tsp

```diff
@@ -437,34 +437,6 @@ model Provider {
   resourceUsageId?: string;
 }
 
-/**
- * Common fields that are returned in the response for all Azure Resource Manager resources
- */
-model Resource {
-  /**
-   * Fully qualified resource ID for the resource. E.g. "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}/{resourceName}"
-   */
-  @visibility("read")
-  id?: Azure.Core.armResourceIdentifier;
-
-  /**
-   * The name of the resource
-   */
-  @visibility("read")
-  name?: string;
-
-  /**
-   * The type of the resource. E.g. "Microsoft.Compute/virtualMachines" or "Microsoft.Storage/storageAccounts"
-   */
-  @visibility("read")
-  type?: string;
-
-  /**
-   * Azure Resource Manager metadata containing createdBy and modifiedBy information.
-   */
-  @visibility("read")
-  systemData?: SystemData;
-}
 
 /**
  * Tags object for patch operations.
```

Comment:
Can you also review the usage of this? We have a standard `ArmTagsPatchAsync|Sync` which doesn't require you to declare this.

Reply:
Fixed, thank you!